Form-Controls/index.html

```diff
@@ -4,25 +4,53 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <title>My form exercise</title>
-  <meta name="description" content="">
+  <meta name="description" content="T-shirt order form">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-  <header>
+  <header role="banner">
     <h1>Product Pick</h1>
   </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <main role="main" aria-labelledby="form-title">
+    <form id="tshirt-order" aria-labelledby="order-title">
+      <h2 id="order-title">Order your T-shirt</h2>
+      
+      <label for="name">Name:</label>
+      <input type="text" id="name" name="name" required pattern="[A-Za-z\s]{2,}" title="Please enter at least 2 characters" aria-label="Name input field">
+      
+      <label for="email">Email:</label>
+      <input type="email" id="email" name="email" required aria-describedby="email-error" />
+      <span id="email-error" class="visually-hidden">Please enter a valid email address (e.g., example@domain.com).</span>
```

Comment:
When you test this form, what happens to these errors? What would the UX be if you were using this form with a screenreader?  A bit confusing?